LDMP/settings.py

```diff
@@ -235,6 +238,15 @@ def __init__(self, parent=None):
         self.populate_cities()
         self.area_admin_0.currentIndexChanged.connect(self.populate_admin_1)
         self.area_admin_0.currentIndexChanged.connect(self.populate_cities)
+        self.area_admin_0.currentIndexChanged.connect(self.generate_name_setting)
+
+        self.secondLevel_area_admin_1.currentIndexChanged.connect(self.generate_name_setting)
```

Comment:
what would be the meaning of "admin_1" in secondLevel_area_admin_1 combobox menu?

Reply:
I'm just reusing the widgets name as defined from the area widget. Do you want us to update them?

Reply:
no... it's ok